package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "gpii-grunt-mdjson-lint",
+  "description": "A plugin to lint JSON and JSON5 blocks within markdown documents.",
+  "version": "1.0.0",
+  "homepage": "",
+  "author": "Tony Atkins <tony@raisingthefloor.org>",
+  "repository": {
+    "type": "git",
+    "url": "git://gihub.com/the-t-in-rtf/grunt-mdjson-linter.git"
+  },
+  "licenses": [
+    {
+      "type": "BSD-3-Clause",
+      "url": "/blob/master/LICENSE-BSD-3-Clause"
+    }
+  ],
+  "engines": {
+    "node": ">= 0.8.0"
```

Comment:
Probably rash to claim that any of our stuff works with <= node 6 these days given we don't test it

Reply:
Fixed.

Reply:
Oops, I see that I need to go in the other direction.  Actually fixed now.